execute-single-test.sh

```diff
@@ -4,10 +4,56 @@ SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
 
 source ${SCRIPT_DIR}/docker/dev.env
 
-TEST_CASE_NAME=$1
+function print_usage() {
+  printf '\n%b\n\n' "Usage: ${0} [ -a APPLICATION_NAME ] [ -s SCHEMA_VERSION ]"
+  printf '%b\n' "Executing a given unit/integration test from the given application using the given schema version"
+  printf '\n%b\n' "-p\tName of the project the test can be found\tfor example: app or atlas-web-core"
+  printf '\n%b\n' "-s\tNumeric version of the schema or latest\tfor example: 18 (default is latest)"
+  printf '%b\n\n' "-h\tShow usage instructions"
```

Comment:
Same suggestions as in `debug-single-test.sh` apply here.

Reply:
Done